.github/workflows/build.yml

```diff
@@ -133,7 +179,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install dependencies
-        run: brew install automake libressl pkg-config krb5 berkeley-db
+        run: brew install automake libressl pkg-config krb5 berkeley-db meson
```

Comment:
Do we still need to install pkg-config here? I think you removed it from the main branch job.